src/Passes/Source/Rules/FactoryConfig.hpp

```diff
@@ -138,8 +143,9 @@ namespace quantum
 
         /// Optimisations
         /// @{
-        bool optimise_branch_quatum_one_{true};
-        bool optimise_branch_quatum_zero_{true};
+        bool optimise_branch_quantum_one_{true};
+        bool optimise_branch_quantum_zero_{true};
+        bool optimise_select_quatnum_zero_{true};
```

Comment:
quatum

Reply:
should be `optimise_select_quantum_zero_` instead of `optimise_select_quatum_zero_`.